ci-operator/step-registry/ipi/install/libvirt/install/ipi-install-libvirt-install-commands.sh

```diff
@@ -123,15 +123,26 @@ for ((i=0; i<${MASTER_REPLICAS}; i++))
 do
   yq write --inplace ${dir}/openshift/99_openshift-cluster-api_master-machines-${i}.yaml spec.providerSpec.value[domainMemory] ${MASTER_MEMORY}
   yq write --inplace ${dir}/openshift/99_openshift-cluster-api_master-machines-${i}.yaml spec.providerSpec.value.volume[volumeSize] ${MASTER_DISK}
-  if [ "${BRANCH}" == "4.9" ] || [ "${BRANCH}" == "4.10" ] && [ "${ARCH}" == "ppc64le" ]; then
-    yq write --inplace ${dir}/openshift/99_openshift-cluster-api_master-machines-${i}.yaml spec.providerSpec.value[Vcpu] 8
+  if [ "${BRANCH}" == "4.9" ] || [ "${BRANCH}" == "4.10" ]; then
+    yq write --inplace ${dir}/openshift/99_openshift-cluster-api_master-machines-${i}.yaml spec.providerSpec.value[domainVcpu] 8
   fi
 done
 # Bump the libvirt workers memory to 16GB
 yq write --inplace ${dir}/openshift/99_openshift-cluster-api_worker-machineset-0.yaml spec.template.spec.providerSpec.value[domainMemory] ${WORKER_MEMORY}
 # Bump the libvirt workers disk to to 30GB
 yq write --inplace ${dir}/openshift/99_openshift-cluster-api_worker-machineset-0.yaml spec.template.spec.providerSpec.value.volume[volumeSize] ${WORKER_DISK}
 
+#if [ "${BRANCH}" == "4.9" ] || [ "${BRANCH}" == "4.10" ] && [ "${ARCH}" == "s390x" ]; then
+ #yq write --inplace ${dir}/openshift/99_openshift-cluster-api_worker-machineset-0.yaml spec.providerSpec.value[domainVcpu] 8
+#fi
+echo "BRANCH = ${BRANCH}"
+echo "ARCH = ${ARCH}"
+if [ "${BRANCH}" == "4.9" ] || [ "${BRANCH}" == "4.10" ] && [ "${ARCH}" == "s390x" ]; then
+  echo "condition met"
+fi
+yq write --inplace ${dir}/openshift/99_openshift-cluster-api_worker-machineset-0.yaml spec.providerSpec.value[domainVcpu] 8
```

Comment:
`spec.template.spec.providerSpec.value.`  for worker node

Reply:
resolved in the recent commit.